<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS><TS version="2.0" language="fr_FR" sourcelanguage="">
<context>
    <name>Photo2Shape</name>
    <message>
        <location filename="photo2shape.py" line="76"/>
        <source>Quantum GIS version detected: </source>
        <translation>Version de QGis détectée :</translation>
    </message>
    <message>
        <location filename="photo2shape.py" line="75"/>
        <source>This version of RasterCalc requires at least QGIS version 1.0.0
Plugin will not be enabled.</source>
        <translation type="obsolete">Cette version de photo2shape a besoin au minimum de QGis 1.0
Le Plugin ne pourra pas être chargé.</translation>
    </message>
    <message>
        <location filename="photo2shape.py" line="83"/>
        <source>Create a point shapefile from a set of geotagged images</source>
        <translation>Crée un fichier shape de type point à partir d&apos;e photos géoréférencées</translation>
    </message>
    <message>
        <location filename="photo2shape.py" line="104"/>
        <source>About Photo2Shape</source>
        <comment>Window title</comment>
        <translation>à propos</translation>
    </message>
    <message>
        <location filename="photo2shape.py" line="106"/>
        <source>&lt;b&gt;Photo2Shape&lt;/b&gt;</source>
        <translation>&lt;b&gt;Photo2Shape&lt;/b&gt;</translation>
    </message>
    <message>
        <location filename="photo2shape.py" line="102"/>
        <source>Version: %1</source>
        <translation type="obsolete">Version: %1</translation>
    </message>
    <message>
        <location filename="photo2shape.py" line="112"/>
        <source>This plugin creates a point shapefile
from a set of geotagged images</source>
        <translation>Ce plugin crée un fichier shape de type point à partir d&apos;e photos géoréférencées</translation>
    </message>
    <message>
        <location filename="photo2shape.py" line="113"/>
        <source>&lt;b&gt;Developers:&lt;/b&gt;</source>
        <translation>&lt;b&gt;Developers:&lt;/b&gt;</translation>
    </message>
    <message>
        <location filename="photo2shape.py" line="116"/>
        <source>&lt;b&gt;Homepage:&lt;/b&gt;</source>
        <translation>&lt;b&gt;Homepage:&lt;/b&gt;</translation>
    </message>
    <message>
        <location filename="photo2shape.py" line="133"/>
        <source>Close</source>
        <translation>Fermer</translation>
    </message>
    <message>
        <location filename="photo2shape.py" line="75"/>
        <source>This version of Photo2Shape requires at least QGIS version 1.0.0
Plugin will not be enabled.</source>
        <translation type="obsolete">Cette version de photo2shape a besoin au minimum de QGis 1.0
Le Plugin ne pourra pas être chargé.</translation>
    </message>
    <message>
        <location filename="photo2shape.py" line="76"/>
        <source>This version of Photo2Shape requires at least QGIS version 2.0.
Plugin will not be enabled.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="photo2shape.py" line="95"/>
        <source>Photo2Shape</source>
        <translation type="unfinished">Photo2Shape</translation>
    </message>
    <message>
        <location filename="photo2shape.py" line="109"/>
        <source>Version: %s</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>Photo2ShapeDialog</name>
    <message>
        <location filename="photo2shapedialog.py" line="207"/>
        <source>Photo2Shape</source>
        <translation>Photo2Shape</translation>
    </message>
    <message>
        <location filename="photo2shapedialogbase.ui" line="20"/>
        <source>Directory with images</source>
        <translation>Répertoire des photos</translation>
    </message>
    <message>
        <location filename="photo2shapedialogbase.ui" line="53"/>
        <source>Browse</source>
        <translation>Parcourir</translation>
    </message>
    <message>
        <location filename="photo2shapedialogbase.ui" line="41"/>
        <source>Output shapefile</source>
        <translation>Fichier shape en sortie</translation>
    </message>
    <message>
        <location filename="photo2shapedialogbase.ui" line="69"/>
        <source>Add result to canvas</source>
        <translation>Afficher le résultat</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="62"/>
        <source>Select directory with images</source>
        <translation>Sélectionner le répertoire des photos</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="75"/>
        <source>No images found</source>
        <translation>Pas de photo à un format adapté trouvée</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="75"/>
        <source>There are no supported images in this directory. Please select another one.</source>
        <translation>Il n&apos;y a pas de photo à un format adapté dans ce répertoire, merci d&apos;en choisir un autre.</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="92"/>
        <source>Select output shapefile</source>
        <translation>Sélectionner le fichier shape en sortie</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="130"/>
        <source>Delete error</source>
        <translation>Erreur dans la destruction</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="119"/>
        <source>Can&apos;t delete file %1</source>
        <translation type="obsolete">Ne peut pas effacer le fichier %1</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="151"/>
        <source>Cancel</source>
        <translation>Annuler</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="167"/>
        <source>There are no geotagged photos in selected directory.
Shapefile was not created.</source>
        <translation>Il n&apos;y a pas de photo géoréférencée dans ce répertoire, merci d&apos;en choisir un autre.</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="197"/>
        <source>Close</source>
        <translation>Fermer</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="183"/>
        <source>Error loading output shapefile:
%1</source>
        <translation type="obsolete">Erreur dans le chargement du fichier shape :
%1</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="221"/>
        <source>QML exists</source>
        <translation>Le fichier QML existe</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="196"/>
        <source>QML file %1 already exists. Overwrite?</source>
        <translation type="obsolete">Le fichier QML %1 existe, l&apos;écraser ?</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="231"/>
        <source>QML error</source>
        <translation>Erreur avec le fichier QML</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="231"/>
        <source>Can&apos;t write QML file</source>
        <translation>Ne peut pas écrire le fichier QML</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="121"/>
        <source>Wrong output file</source>
        <translation>Fichier en sotie faux</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="121"/>
        <source>Output file is improperly defined.
Please enter a valid filename and try again.</source>
        <translation>Fichier en sortie mal défini.
Merci d&apos;entrer un nom valide et de réessayer.</translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="90"/>
        <source>Shapefiles (*.shp *.SHP)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="130"/>
        <source>Can&apos;t delete file %s</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="175"/>
        <source>The following files were not added to shapefile because of errors: &lt;br&gt;&lt;br&gt;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="207"/>
        <source>Error loading output shapefile:
%s</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="photo2shapedialog.py" line="221"/>
        <source>QML file %s already exists. Overwrite?</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
